DirectEmployers/MyJobs
# Hide Partner Assessment link for deleted partners

## What's this PR do?
Hide the "Partner Assessment" link for deleted partners
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
In PRM, Delete a partner and then check the "Include Deleted Partners". click on the deleted partner. The "Partner Assessment: link shouldn't be visible.

## Any background context you want to provide?
